tools/mmp/driver.cs

```diff
@@ -76,6 +76,7 @@ public static partial class Driver {
 		static List<string> resolved_assemblies = new List<string> ();
 		static List<string> ignored_assemblies = new List<string> ();
 		static List<string> native_references = new List<string> ();
+		static List<string> binding_project_no_embedding = new List<string> ();
```

Comment:
nothing use this inside the commit ?

Reply:
I missed it when I hoisted it to the common Application. Good catch